go_test.go

```diff
@@ -43,7 +43,7 @@ func TestGoBuildInfo(t *testing.T) {
 	// 3. Validate the total count of dependencies added to the build-info.
 	buildNumber := "1"
 
-	err = execGo(t, artifactoryCli, "go", "build", tests.GoRepo, "--build-name="+tests.GoBuildName, "--build-number="+buildNumber)
+	err = execGo(t, artifactoryCli, "go", "build --mod=mod", tests.GoRepo, "--build-name="+tests.GoBuildName, "--build-number="+buildNumber)
```

Comment:
"build --mod=mod" --> "build",  "--mod=mod"

(merging two args into one can cause parsing issues)

Reply:
Since the legacy Go command parses the flags, it will fail with the error "-mod is not supported flag". this workaround should hide the build flag and run the legacy build command successfully.